cachito/workers/config.py

```diff
@@ -93,6 +93,8 @@ class Config(object):
     # other tasks aren't starved when processing a large archive.
     worker_prefetch_multiplier = 1
 
+    subprocess_timeout = 3600  # 1 hour
```

Comment:
```suggestion
    cachito_subprocess_timeout = 3600  # 1 hour
```
The cachito-specific options should have the `cachito_` prefix, the ones that don't have a prefix are celery options I think